tools/modules/framework/interface_message_box.sh

```
@@ -0,0 +1,30 @@
+	["interface_message,group"]="Interface"
+)
+#
+# Function to display a message box
+#
+function interface_message() {
+	# Read the input from the pipe
+	input=$(cat)
+	# Display menu based on DIALOG tool
+	case $DIALOG in
+		"dialog")
+		$DIALOG --title "$TITLE" --msgbox "$input" 0 0 2>&1 >/dev/tty
```

Shellcheck (0.10.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)